### PR TITLE
Add section for outgoing calls and CallKit

### PR DESCRIPTION
**Pull request by GiovanniBeriSinch**

### CODE REVIEW

File: docs/voice/voice-ios-cloud/voice-ios-cloud-push-notifications-callkit.md [hunk 1]
```diff
@@ -126,9 +126,43 @@ When you relay the push notification to a `SINClient`. If you for some reason do
...
+  id<SINCall> call = [_client.callClient callUserWithId: action.handle.value];
+  [action fulfill];
+}
+
+// (For a more complete example, see the Sinch sample app SinchCallKit.xcodeproj)
+```
+
 ### Extracting Call Information From a Push Payload
 
-At the time when your application receives a push notification you will need to extract some key information about the call based on only the push notification payload. You will need to do this to conform with with Apple requirements on reporting a VoIP push notification as an incoming call to _CallKit_, but you may also want to extract application-specific headers for the call. Use the method `+[SINManagedPush queryPushNotificationPayload:]` to extract call details from the raw push payload. Note that you can do this immediately and before you have created and started a `SINClient` instance.
```
Comment (GiovanniBeriSinch) [inline docs/voice/voice-ios-cloud/voice-ios-cloud-push-notifications-callkit.md:131]: here I removed "with with Apple requirements". It's not easy to see diffs in these long lines :)

File: docs/voice/voice-ios-cloud/voice-ios-cloud-push-notifications-callkit.md [hunk 2]
```diff
@@ -126,9 +126,43 @@ When you relay the push notification to a `SINClient`. If you for some reason do
...
+
+```objectivec
+
+// (Assuming CallKit CXCallController accessible as self.callController)
+- (void)startOutgoingCall:(NSString *)destination {
+  NSUUID *callId = [[NSUUID alloc] init];
+
+  CXHandle *handle = [[CXHandle alloc] initWithType:CXHandleTypeGeneric value:destination];
+  CXStartCallAction *initiateCallAction =
+      [[CXStartCallAction alloc] initWithCallUUID:callId handle:handle];
```
Comment (dmipon) [inline docs/voice/voice-ios-cloud/voice-ios-cloud-push-notifications-callkit.md:144]: Should we actually have a line break here where rest of the text much longer ?

**Comment by dmipon** [comment]
LGTM except one little comment.

File: docs/voice/voice-ios-cloud/voice-ios-cloud-push-notifications-callkit.md (hunk 2 above)
Comment (GiovanniBeriSinch) [reply docs/voice/voice-ios-cloud/voice-ios-cloud-push-notifications-callkit.md:144]: I think it still improves readability... matter of taste I guess

**Comment by dmipon** [comment]
:ship: it

**Pull request merged**